lib/facebook_data_analyzer.rb

```diff
@@ -2,21 +2,21 @@
 
 # My script for 'I analyzed my facebook data and it's story of shyness,
 # loneliness and change'
-module FacebookDataAnalyzer
-  require 'facebook_data_analyzer/analyzeables/analyzeable'
-  require 'facebook_data_analyzer/analyzeables/contacts'
-  require 'facebook_data_analyzer/analyzeables/friends'
-  require 'facebook_data_analyzer/analyzeables/messages'
-  require 'facebook_data_analyzer/contact'
-  require 'facebook_data_analyzer/friend'
-  require 'facebook_data_analyzer/message'
-
-  require 'axlsx'
-  require 'parallel'
-  require 'json'
-  require 'workbook'
-  require 'set'
+require 'axlsx'
```

Comment:
Out of curiosity, why is that moved outside of the `module FacebookDataAnalyzer` - is there some best practices?

Probably line 26 can be changed from `package = ::Axlsx::Package.new` to `package = Axlsx::Package.new`.
 Same with `b = ::Workbook::Book.open(xlsx)` on line 41

Reply:
Yeah, it is quite common  to do so. For instance: https://github.com/sferik/twitter/blob/master/lib/twitter.rb (I  often use this gem as a reference because it was  written by a team of very experienced guys).

Reply:
To elaborate, there is no reason to `require` inside a module, because the required constants (including classes/modules which are constants too) will be available globally anyways.

Reply:
@thnukid: I will update lines 26 and 41, as you suggested.  
I tracked line 26 to: 7f1b8d09 (Eric Bartholemy 2018-05-05 00:04:50 +0200 26)     package = ::Axlsx::Package.new, so I can safely assume the contributor will not mind. :)
I cannot easily track the origin of current line 41, due to file renames and frequent commits, but will update it as well, for consistency.

Reply:
Good that I have not left any tracks on line 41 :D

Reply:
Lol! No worries! This is a good update.